### PR TITLE
Add code styling

### PR DESCRIPTION
**Pull request by santiagosemhan**
This PR:

- Adds Airbnb code style.
- Removes unused files.
- Refactor endpoints. 
- Set the basis for lint GitHub action on each PR.

### CODE REVIEW

File: src/server.js [hunk 1]
```diff
@@ -27,15 +27,9 @@ app.use('/public', express.static(path.join(__dirname, '/public')));
 // Serve the Parse API on the /parse URL prefix
 app.use(PARSE_MOUNT || '/parse', api);
 
-// Parse Server plays nicely with the rest of your web routes
+// health endpoint
 app.get('/', async (request, response) =>
```
Comment (Benjamindodgson) [inline src/server.js:31]: What is this?

File: src/services/ReservationService.js [hunk 2]
```diff
@@ -15,7 +15,7 @@ export class ReservationServiceError extends ExtendableError {}
 const createReservation = async user => {
   try {
     const reservation = new Parse.Object('Reservation');
-    const reservationCount = await db.getValueForNextSequence('reservation');
+    // const reservationCount = await db.getValueForNextSequence('reservation');
```
Comment (Benjamindodgson) [inline src/services/ReservationService.js:18]: Should this be deleted?

File: src/utils/db.js [hunk 3]
```diff
@@ -16,18 +16,19 @@ function getDatabaseInstance() {
...
+    const sequences = db.collection('_sequences'); // returns new instance of _sequences if collections doesn't exists.
+    const { value } = await sequences.findOneAndUpdate(
       { _id: sequenceOfName },
       { $inc: { sequence_value: 1 } },
       { upsert: true },
     );
     return !value ? value + 1 : value.sequence_value + 1;
   } catch (error) {
-    console.log('db error');
+    // console.log('db error');
```
Comment (Benjamindodgson) [inline src/utils/db.js:31]: delete?

**Review by Benjamindodgson — APPROVED** [review]
looks good. just a few questions

File: src/server.js (hunk 1 above)
Comment (santiagosemhan) [reply src/server.js:31]: I removed the example endpoint and instead created an endpoint you could query to check if the server is online.

File: src/services/ReservationService.js (hunk 2 above)
Comment (santiagosemhan) [reply src/services/ReservationService.js:18]: You are right

**Comment by santiagosemhan** [comment]
I'll merge this PR and automate lint check on future PR's

**Pull request closed without merging**